TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/206_Motion_Vectors/CaptureMotionVectorsRendererFeature.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+internal class CaptureMotionVectorsRendererFeature : ScriptableRendererFeature
+{
+    public Shader m_Shader;
+    public float m_Intensity;
+
+    Material m_Material;
+
+    CaptureMotionVectorsPass m_RenderPass = null;
+
+    public override void AddRenderPasses(ScriptableRenderer renderer, ref RenderingData renderingData)
+    {
+        if (renderingData.cameraData.cameraType == CameraType.Game)
+        {
+            m_RenderPass.ConfigureInput(ScriptableRenderPassInput.Motion);
+            m_RenderPass.SetTarget(renderer.cameraColorTarget, m_Intensity);
+            renderer.EnqueuePass(m_RenderPass);
+        }
+    }
+
+    public override void Create()
+    {
+        if (m_Shader != null)
+            m_Material = new Material(m_Shader);
+        else
+            Debug.LogError(typeof(CaptureMotionVectorsRendererFeature) + " was assigned an invalid shader.");
```

Comment:
This error triggers when loading the Graphics\TestProjects\UniversalGraphicsTest_Foundation project for some reason. It triggers twice on project load, but not afterwards nor when opening/running the scene.

![image](https://user-images.githubusercontent.com/72749964/117840933-ddb13880-b27c-11eb-867f-2b58a0581507.png)

Reply:
Looking at other tests, I am sure this same problem happens, but the other tests do not assert. I think it would be fair just to remove the debug assert and follow-up on this later.

Reply:
@ellioman have you hit this issue in your test scenes?

Reply:
I have not, I believe it's because I do a `Shader.Find()` call for the shader if it's null in SSAO.

But I've been meaning to check out changing that so the shader reference would be stored in the `ShaderResources` class in `UniversalRenderPipelineEditorResources.cs`and get it from `UniversalRenderPipelineAsset.cs` like we already do with shaders like terrain, speedtree, autodesk, etc.

Reply:
I'll remove the error message. It's not really needed.